manim/mobject/svg/tex_mobject.py

```diff
@@ -26,13 +28,7 @@
 TEX_MOB_SCALE_FACTOR = 0.05
 
 
-class TexSymbol(VMobjectFromSVGPathstring):
```

Comment:
Did this go away?

Reply:
Yup. It was just a renaming, and was only used in _one_ place in the same file.